File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] numbers, String direction) {
    int[] answer = new int[numbers.length * 2];
    System.arraycopy(numbers, 0, answer, 0, numbers.length);
    System.arraycopy(numbers, 0, answer, numbers.length, numbers.length);
    if (direction.equals("right")){
        return Arrays.copyOfRange(answer, numbers.length - 1, numbers.length*2 - 1);
    }

    else {
        return Arrays.copyOfRange(answer, 1, numbers.length + 1);
    }
}
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.go
func solution(numbers []int, direction string) []int {
    answer := []int{}
    n := len(numbers)

    if direction == "right" {
        answer = append([]int{numbers[n-1]}, numbers[:n-1]...)
    } else {
        answer = append(numbers[1:], numbers[0])
    }
    return answer
}